pkg/ui/src/FormFields/Checkbox/Checkbox.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+	export let name: string;
+
+	export let label = '';
+	export let helpText = '';
+	export let checked = false;
+	export let disabled = false;
+
+	const labelBaseClasses = ['font-medium', disabled ? 'text-gray-400' : 'text-gray-700'].join(' ');
+	const helpTextColor = disabled ? 'text-gray-300' : 'text-gray-500';
+	const inputBaseClasses = [
+		'focus:ring-teal-500',
+		'h-4',
+		'w-4',
+		'text-gray-800',
+		'border-gray-300',
+		'rounded',
+		'disabled:text-gray-200'
+	].join(' ');
+</script>
+
+<div class="relative flex items-start">
+	<div class="flex items-center h-5">
+		<input
+			type="checkbox"
+			id={name}
```

Comment:
The `id` of an HTML element needs to be unique in the page. Using the `name` could lead to problems.
Say we have a boolean field `available` in a book form. If we ever use two instances of the book form on the page, there will be two checkboxes with the same id. I think (I'm not sure) in that case all labels would drive the first checkbox.
To solve the same problem in Eisbuk we created a unique id on the fly at component creation, and use that in the `checkbox` `id` and the `label` `for` attributes.

Reply:
Updated to use uuid for checkbox id